scripts/globals/abilities/chocobo_jig_ii.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- Ability: Chocobo jig ii
+-- Increases Movement Speed to party members within the area of effect.
+-- Obtained: Dancer Level 70
+-- TP Required: 0
+-- Recast Time: 1:00
+-- Duration: 2:00
+-----------------------------------
+ 
+require("scripts/globals/settings");
+require("scripts/globals/status");
+
+-----------------------------------
+-- onAbilityCheck
```

Comment:
please stop adding these function headers that do nothing but repeat the function names, I stripped them all out awhile back and with ppl adding them again we'll have to do it again